### PR TITLE
MAUI-3166 : Added API reference links for the MAUI Maps Sublayer UG

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @vigneshrameshsync ,
I added a API reference links for the MAUI Maps Sublayer UG. Could you please review it?

Regards,
Shalini Ashokan.

### CODE REVIEW

**Comment by Shalini-Ashokan** [comment]
rebuild

**Comment by vigneshrameshsync** [comment]
rebuild

**Comment by vigneshrameshsync** [comment]
rebuild

File: MAUI/Maps/shape-sublayer.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ N> Refer the `MapShapeLayer`, for adding shape layer in `SfMaps`.
 
 ## Color and stroke color
```
Comment (vigneshrameshsync) [inline MAUI/Maps/shape-sublayer.md:57]: The note section in the above line, API reference is not included.

N> Refer the `MapShapeLayer`, for adding shape layer in `SfMaps`.

File: MAUI/Maps/vector-layers/arc-layer.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ this.Content = maps;
 
 The `HeightFactor` is the distance from the line connecting two points to the arc bend point. The default value of `HeightFactor` property is `0.2` and the value ranges from -1 to 1.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/vector-layers/arc-layer.md:127]: API reference not included.

File: MAUI/Maps/vector-layers/circle-layer.md [hunk 3]
```diff
@@ -9,11 +9,11 @@ documentation: ug
...
+Circle layer is a sublayer that renders a group of [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) on [`MapShapeLayer`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html). This section helps to learn about how to add the circles and customize them.
 
 ## Adding circles
 
-The `Circles` is a collection of `MapCircle`. Every single `MapCircle` renders a circle using the `MapCircle.Center` and `MapCircle.Radius` properties.
+The [`Circles](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircleLayer.html#Syncfusion_Maui_Maps_MapCircleLayer_Circles) is a collection of [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html). Every single [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) renders a circle using the [`MapCircle.Center`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html#Syncfusion_Maui_Maps_MapCircle_Center) and [`MapCircle.Radius`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html#Syncfusion_Maui_Maps_MapCircle_Radius) properties.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/vector-layers/circle-layer.md:16]: Syntax error

File: MAUI/Maps/vector-layers/circle-layer.md [hunk 4]
```diff
@@ -228,7 +228,7 @@ this.Content = maps;
 
 ## Fill color
 
-You can apply colors to each `MapCircle` in the `Circles` collection using the individual `MapCircle.Fill` property.
+You can apply colors to each [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) in the [`Circles](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircleLayer.html#Syncfusion_Maui_Maps_MapCircleLayer_Circles) collection using the individual [`MapCircle.Fill`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html#Syncfusion_Maui_Maps_MapCircle_Fill) property.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/vector-layers/circle-layer.md:231]: Syntax error for `Circles`

File: MAUI/Maps/vector-layers/circle-layer.md [hunk 5]
```diff
@@ -353,9 +353,9 @@ this.Content = maps;
 
 ## Stroke and stroke thickness
 
-You can apply stroke thickness to each `MapCircle` in the `Circles` collection using the individual `MapCircle.StrokeThickness` property. The default value of the `MapCircleLayer.StrokeThickness` property is `2`.
+You can apply stroke thickness to each [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) in the [`Circles](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircleLayer.html#Syncfusion_Maui_Maps_MapCircleLayer_Circles) collection using the individual [`MapCircle.StrokeThickness`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html#Syncfusion_Maui_Maps_MapCircle_StrokeThickness) property. The default value of the [`MapCircle.StrokeThickness`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html#Syncfusion_Maui_Maps_MapCircle_StrokeThickness) property is `2`.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/vector-layers/circle-layer.md:356]: Syntax error for `Circles`

File: MAUI/Maps/vector-layers/line-layer.md [hunk 6]
```diff
@@ -328,7 +328,7 @@ this.Content = maps;
 
 ## Stroke cap
 
-You can apply stroke cap to each `MapLine` in the `Lines` collection using the individual `MapLine.StrokeLineCap` property. The default value of the `MapLineLayer.StrokeLineCap` property is `StrokeLineCap.Butt`. The available values are `Butt`, `Round`, and `Square`.
+You can apply stroke cap to each [`MapLine`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLine.html) in the [`Lines`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLineLayer.html#Syncfusion_Maui_Maps_MapLineLayer_LinesProperty) collection using the individual [`MapLine.StrokeLineCap`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLine.html#Syncfusion_Maui_Maps_MapLine_StrokeLineCap) property. The default value of the [`MapLine.StrokeLineCap`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLine.html#Syncfusion_Maui_Maps_MapLine_StrokeLineCap) property is `StrokeLineCap.Butt`. The available values are `Butt`, `Round`, and `Square`.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/vector-layers/line-layer.md:331]: API reference not included for `StrokeLineCap.Butt`, `Butt`, `Round`, and `Square`.

File: MAUI/Maps/vector-layers/polyline-layer.md [hunk 7]
```diff
@@ -255,7 +255,7 @@ this.Content = maps;
 
 ## Stroke cap
 
-You can apply different stroke cap to each `MapPolyline` in the `Polylines` collection using the individual `MapPolyline.StrokeLineCap` property. The default value of the `MapPolylineLayer.StrokeLineCap` property is `StrokeLineCap.Butt`. The available values are `Butt`, `Round`, and `Square`.
+You can apply different stroke cap to each [`MapPolyline`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html) in the [`Polylines`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolylineLayer.html#Syncfusion_Maui_Maps_MapPolylineLayer_Polylines) collection using the individual [`MapPolyline.StrokeLineCap`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html#Syncfusion_Maui_Maps_MapPolyline_StrokeLineCap) property. The default value of the [`MapPolylineLayer.StrokeLineCap`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html#Syncfusion_Maui_Maps_MapPolyline_StrokeLineCap) property is `StrokeLineCap.Butt`. The available values are `Butt`, `Round`, and `Square`.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/vector-layers/polyline-layer.md:258]: API reference not included for `StrokeLineCap.Butt`, `Butt`, `Round`, and `Square`.

File: MAUI/Maps/vector-layers/arc-layer.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/arc-layer.md]: Updated the API reference

File: MAUI/Maps/vector-layers/circle-layer.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/circle-layer.md]: Updated the syntax

File: MAUI/Maps/vector-layers/circle-layer.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/circle-layer.md]: Updated the syntax

File: MAUI/Maps/vector-layers/circle-layer.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/circle-layer.md]: Updated the syntax

File: MAUI/Maps/vector-layers/polyline-layer.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/polyline-layer.md]: It's a framework class. So not add any API reference.

File: MAUI/Maps/vector-layers/line-layer.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/line-layer.md]: It's a framework class. So not add any API reference.

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

**Pull request merged**